Prepare Release - v1.9.0 (#205)
1Password/onepassword-operator: CHANGELOG.md

---

[//]: # (START/v1.9.0)
# v1.9.0

## Features

>   * Operator can be used with 1Password Service Accounts now. {#160}

Reviewer: `[nit]`

```suggestion
  * Enable the Operator to authenticate to 1Password using service accounts. {#160}
```
or
```suggestion
  * Add service account support for the Operator. {#160}
```

Using the word `now` in a changelog message is not that elegant.

## Fixes

>  * Update Operator to use SDK v1.34.1. {#185}

Reviewer: `[non-blocking suggestion]`

This happened so long ago that I'm considering doing another upgrade. The latest is `1.40.0` ([link](https://sdk.operatorframework.io/docs/upgrading-sdk-version/)) and while this is an additive element, we should keep our operator up-to-date as a good practice.

Reviewer 2: Good point. Created [an issue](https://github.com/1Password/onepassword-operator/issues/207) to track it.
